docs/installation.md

```diff
@@ -13,15 +13,28 @@
 ## Build Application
 
 ### Download Repository
-Download or clone the repository source code to your workstation.
 
-### Build docker image
+Download or clone the repository source code to your workstation.  
+![Github Clone Section](graphics/clonerepo.png)
+
 
-Open terminal in the project root path where docker-compose.yml is located and execute: 
+* Trough terminal:
 ```bash
-docker-compose build
+git clone https://github.com/industrial-edge/hello-world.git
 ```
-This command builds the docker image of the service which is specified in the docker-compose.yml file.
+
+* Trough VSCode:  
+<kbd>CTRL</kbd>+<kbd>&uarr; SHIFT</kbd>+<kbd>P</kbd> or <kbd>F1</kbd> to open VSCode's command pallette and type `git clone`:
+
+![VS Code Git Clone command](graphics/git.png)
+
+### Build docker image
+
+- Navigate into `src` and find the file named `Dockerfile.example`. The `Dockerfile.example` is an example Dockerfile that can be used to build the Docker images of the backend and the frontend. If you choose to use these, rename them both to `Dockerfile` before proceeding
```

Comment:
There is no backend in this example, please remove this hint here.

Reply:
ah yes of course, I used this template for other repos I have updated as well, will updated accordingly one by one and push right away